<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Line_Loader_Widget</class>
 <widget class="QWidget" name="Line_Loader_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="HDF5LineLoader_Widget" name="hdf5_line_loader" native="true"/>
   </item>
   <item>
    <widget class="CSVLineLoader_Widget" name="csv_line_loader" native="true"/>
   </item>
   <item>
    <widget class="LMDBLineLoader_Widget" name="lmdb_line_loader" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="data_name_text">
         <property name="text">
          <string>line_1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Line Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Scaling_Widget" name="scaling_widget" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HDF5LineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/LineLoaderWidget/HDF5LineLoader_Widget/HDF5LineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSVLineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/LineLoaderWidget/CSVLineLoader_Widget/CSVLineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LMDBLineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/LineLoaderWidget/LMDBLineLoader_Widget/LMDBLineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Scaling_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/Scaling_Widget/Scaling_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
